docs/RPC Encryption/index.md

```diff
@@ -0,0 +1,129 @@
+# RPC Encryption
+
+#### Related Evolution Proposals
+- [0207: RPC Message Protection](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0207-rpc-message-protection.md)
+
+## Overview
+
+The RPC encryption feature is designed to protect RPC messages transmitted between a mobile application and SDL. RPCs that need encryption and RPCs that do not need encryption share the same RPC service with encryption enabled.
+
+## Determining when Encryption is Needed
+
+### SDL Application
+
+An SDL app is informed about the encryption requirement by an `OnPermissionsChange` notification sent by SDL Core. The `OnPermissionsChange` notification has a `requireEncryption` boolean parameter, which indicates the encryption requirement on the app level.
```

Comment:
```suggestion
An SDL app is informed about SDL Core's encryption requirements by an `OnPermissionsChange` notification. The `OnPermissionsChange` notification has a `requireEncryption` boolean parameter, which indicates the encryption requirement for each RPC.
```

Reply:
The `OnPermissionsChange` notification has its own `requireEncryption` param which indicates whether or not the app requires encryption. The RPC level `requireEncryption` is in the list of PermissionItems. From the spec 

```
.. add two new Boolean parameters with the same name `requireEncryption`. One for app level is added to OnPermissionsChange and the other for RPC level is added to the PermissionItem data type
```

Reply:
Ok I guess my main concern was the use of the word "app level". I don't think it is descriptive enough for what it is trying to say. 

> The `OnPermissionsChange` notification has a `requireEncryption` boolean parameter, which indicates the encryption requirement for the App. If `requireEncryption` is false, then no RPC's will require encryption. If `requireEncryption` is true, RPC encryption will be required for any RPC that has `requireEncryption = true` in its `PermissionItem` struct.